lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -16452,3 +16452,9 @@ msgstr "Votre mot de passe expirera dans %s jours."
 
 msgid "Security policy not found. Please verify that your installation is valid"
 msgstr "Politique de sécurité non trouvée. Veuillez vérifier que votre installation est valide"
+
+msgid "Your autologin key should be different than your current password"
```

Comment:
Simple comment here, but I guess the "should" implies that it is not possible to set an autologin key = password. Therefore "should" could be replaced by "must" as it is a requirement